packages/suite-native-navigation-root/src/dummyScreens/DemoScreen.tsx

```diff
@@ -23,6 +25,16 @@ const backgroundStyle = prepareNativeStyle<{ isDarkMode: boolean }>(
     }),
 );
 
+const typographyItems: TypographyStyle[] = [
```

Comment:
I don't care too much about this file since it's just demo for us, but you can import `typographyStylesBase` from theme and then do something like `Object.keys(typographyStylesBase)`... but I will leave it up to you

Reply:
This was done just so the modal is not empty. It will be deleted once we start to populate the sections so as you say, I wouldn't worry about it too much.